src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -2539,6 +2539,34 @@ select query = do
     conn <- R.ask
     liftIO $ with res $ flip R.runReaderT conn . runSource
 
+-- | Execute an @esqueleto@ @SELECT@ query inside @persistent@'s
+-- 'SqlPersistT' monad and return the first entry wrapped in a @Maybe@.
+--
+-- === __Example usage__
+--
+-- @
+-- firstPerson :: MonadIO m => SqlPersistT m (Maybe Person)
+-- firstPerson =
+--  'selectFirst' $
+--      'from' $ \person ->
+--          return person
```

Comment:
Same as below - let's use the `Experimental` syntax instead, since all code will be switching to it.